packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -120,7 +152,7 @@ function verifyRequestId(payload: CallApiPayload): LogsData<ApiCallErrorResponse
 
 export function verifyTemplateId(payload: CallApiPayload): LogsData<ApiCallErrorResponse> | null {
   const { aggregatedApiCall } = payload;
-  if (aggregatedApiCall.type === 'testing-gateway') return null;
+  if (aggregatedApiCall.type !== 'regular') return null;
```

Comment:
This will need to be updated if signed relayed requests will be based on templates (currently it's required to provide all of the parameters when doing the HTTP request).